src/kirby/components/list/list.component.tns.html

```diff
@@ -45,7 +45,7 @@
 
     <!-- FOOTER -->
     <ng-template tkListViewFooter *ngIf="!isLoadOnDemandEnabled && noMoreItemsText && !listFooterTemplate">
-      <Label class="footer" text="{{noMoreItemsText}}"></Label>
+        <Label class="show-more-items" text="{{noMoreItemsText}}"></Label>
```

Comment:
A bit nitpicking here 😄 

But should the class not called something `no-more-items`  like it is on web?